shims/cdpdc71/driver/pom.xml

```diff
@@ -0,0 +1,332 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.pentaho.hadoop.shims</groupId>
+    <artifactId>pentaho-hadoop-shims-cdpdc71-reactor</artifactId>
+    <version>9.2.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pentaho-hadoop-shims-cdpdc71-driver</artifactId>
+  <version>9.2.0.0-SNAPSHOT</version>
+  <packaging>bundle</packaging>
+
+  <properties>
+    <commons-lang.version>2.6</commons-lang.version>
+    <pentaho-osgi-bundles.version>9.2.0.0-SNAPSHOT</pentaho-osgi-bundles.version>
+    <org.apache.avro.version>1.8.0</org.apache.avro.version>
+    <org.apache.orc.version>1.2.3</org.apache.orc.version>
+    <org.apache.hadoop.version>3.1.1.7.1.3.0-100</org.apache.hadoop.version>
+    <org.apache.hbase.version>2.2.3.7.1.3.0-100</org.apache.hbase.version>
+    <org.apache.hive.version>2.1.1-cdh6.1.0</org.apache.hive.version>
+    <org.apache.oozie.version>5.1.0.7.1.3.0-100</org.apache.oozie.version>
+    <pig.version>0.17.0-cdh6.1.0</pig.version>
```

Comment:
There are no pig components in the CDPDC7.1 cluster, should we keep it as it is ?